packages/x-components/src/x-modules/queries-preview/components/query-preview.vue

```diff
@@ -2,12 +2,14 @@
   <NoElement v-if="queryPreviewResults && queryPreviewResults.totalResults">
     <!--
       @slot Query Preview default slot.
-          @binding {string} query - query
+          @binding {QueryPreviewInfo} queryPreviewInfo - The information about the request of the
+          query preview
+          @binding {Dictionary<unknown>} injectedParams - The extra params from the snippetConfig
```

Comment:
params are being bound inside QueryPreviewInfo one, so no need for this doc here